### DISCUSSION TITLE
WiFi | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#811] [Q&A]
# WiFi | Seeed Studio Wiki

WiFi Usage on Seeed Studio XIAO ESP32C3

https://wiki.seeedstudio.com/XIAO_ESP32C3_WiFi_Usage/

**Comment by wstaniszewski** [comment]
I DOES NOT Work with WIFI.   Scan for network work fine. But never connects to wifi with password.

**Reply by JasonGodlove** [reply]
I was having a similar issue, constantly getting Unknown Failure 2.
That is until I did Step 1 and Connected the included WiFi/ Bluetooth antenna. After that my Seeed Studio XIAO ESP32C3 connected on the first try and I was able to ping google.
